ci/buildkite.py

```diff
@@ -86,11 +87,16 @@ def cmd_build(args, remainder):
     common_args += ['--verbose_failures']
     common_args += ['--verbose_explanations']
     if platform.system() == 'Windows':
-        # TODO: Test everything on windows
         util.check_call(['git', 'config', 'core.symlinks', 'true'])
-        util.check_call(['bazelisk', 'build', ':pkg'] + common_args)
+        cenv = util.CondaEnv(pathlib.Path('.cenv'))
+        cenv.create('environment-windows.yml')
+        env = os.environ.copy()
+        env.update(cenv.env())
+        env['PLAIDML_EXPERIMENTAL'] = '1'
```

Comment:
I think this should be part of the `~/.plaidml` file that's installed on the build agent.

Reply:
That was an experiment to see if it fixed the build -- it didn't.  I installed the Intel OpenCL CPU drivers on the Windows build machines, and it's working fine now.